requirements-dev.txt

```diff
@@ -3,7 +3,7 @@ matplotlib>=2.1
 black[jupyter]~=22.0
 pylint>=2.16.2
 pylatexenc>=1.4
-stestr>=2.0.0
+stestr>=4.1.0
```

Comment:
I guess this is fine - the constraint change that Qiskit did was simply closed. CI pulls in the latest version anyway even with >=2.0.0, but if you had some local older stestr install working this requires things to be updated if you pip install this dev file. Qiskit left their requirements-dev as it was before. I guess if you had things working locally and somehow   just updated python-subunit then it would faili if you did not have anything installed then as of now, like CI, it would install 4.1.0 and work.

Reply:
Yes, I updated the version because someone may update only python-subunit and have the stestr trouble.